stdlib/2/typing.pyi

```diff
@@ -16,6 +16,7 @@ Type = object()
 _promote = object()
 no_type_check = object()
 ClassVar = object()
+Protocol = object()
```

Comment:
This form is actually necessary (in some sense) since mypy will prohibit any other operations with ``Protocol`` in runtime context, like:
```python
Protocol()  # instantiating is a runtime error
Protocol[int]  # type application is a runtime error
def make_one(cls: type) -> Any: ...
make_one(Protocol)  # passing to a function is not a good idea
```
This could be fixed by additional special-casing, but a simpler way is just make it an ``object()``.